RFS/src/test/java/com/rfs/common/LuceneDocumentsReaderTest.java

```diff
@@ -202,7 +202,7 @@ protected DirectoryReader wrapReader(DirectoryReader reader, boolean softDeletes
 
             // Verify results
             var expectedConcurrentSegments = 5;
-            var expectedConcurrentDocReads = Schedulers.DEFAULT_BOUNDED_ELASTIC_SIZE -  Schedulers.DEFAULT_BOUNDED_ELASTIC_SIZE % expectedConcurrentSegments;
+            var expectedConcurrentDocReads = 100;
```

Comment:
Updated this value since I changed all these constants in the LuceneDocumentReader